Alloy/commands/compile/ast/handle-alloy-globals.js

```diff
@@ -0,0 +1,98 @@
+const toCheck = [ 'Alloy', '_', 'Backbone' ];
+var template = require('@babel/template').default;
```

Comment:
If you're already using `const` above, why not also use `const` or `let` in place of these `var`s?